src/test/resources/com/atomist/rug/test/gherkin/handler/event/PassingFeature2Steps1.ts

```diff
@@ -0,0 +1,17 @@
+import {Given,When,Then, HandlerScenarioWorld} from "@atomist/rug/test/handler/Core"
+import * as node from "../../../handlers/event/Nodes"
+
+When("i call you", world => {
+   world.registerHandler("ReturnsAMessage")
+   world.registerHandler("AngryHandler")
+   world.sendEvent(new node.Commit)
+   world.sendEvent(new node.Commit)
+   world.sendEvent(new node.GitHubId("myId"))
+})
+Then("you call me", world => {
+    return world.message() != null
+})
+Then("to greet me", world => {
+    let message = world.message()
```

Comment:
Shouldn't this be gone if we can't return a Message on its own (I mean not in a plan).